src/main/java/roomescape/auth/AuthAdminInteceptor.java

```diff
@@ -0,0 +1,31 @@
+package roomescape.auth;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.HandlerInterceptor;
+import roomescape.infrastructure.JwtTokenUtil;
+import roomescape.member.Member;
+import roomescape.member.MemberService;
+
+@Component
+public class AuthAdminInteceptor implements HandlerInterceptor {
+    private MemberService memberService;
+    private JwtTokenUtil jwtTokenUtil;
+
+    public AuthAdminInteceptor(MemberService memberService, JwtTokenUtil jwtTokenUtil) {
+        this.memberService = memberService;
+        this.jwtTokenUtil = jwtTokenUtil;
+    }
```

Comment:
`@Autowired` 어노테이션을 사용하여 의존성 주입을 명확하게 하는 것도 가독성에 좋을 거 같네요!

```suggestion
    @Autowired
    public AuthAdminInteceptor(MemberService memberService, JwtTokenUtil jwtTokenUtil) {
        this.memberService = memberService;
        this.jwtTokenUtil = jwtTokenUtil;
    
```